packages/cli/src/commands/start.ts

```diff
@@ -15,8 +16,8 @@ export const start: StartHandler = async (filename: string, verbose: boolean) =>
   }
 
   filename = path.isAbsolute(filename) ? filename : path.join(process.cwd(), filename);
-
-  if (!existsSync(filename)) {
+  const isFileExist = await promisify(exists)(filename);
```

Comment:
promisify should be called statically.

Reply:
Maybe we should use `fs-extra` that supported promise-based function instead since it as a dependency of `cli`